src/components/careEntry/index.tsx

```diff
@@ -0,0 +1,206 @@
+  padding: 0px 10px;
+  line-height: 0px;
+`;
+
+const StyledClose = styled(CloseOutlined)`
+  color: red;
+  padding: 5px;
+  border-radius: 3px;
+
+  & :hover {
+    background-color: #ffd1d1;
+  }
+`;
+
+const DeleteActivityButton = styled(LinkButton)`
+  color: white;
+  margin: 10px;
+  padding: 0px 10px;
+  background: ${LIGHT_RED};
+  border: none;
+  &:hover {
+    color: ${LIGHT_RED};
+    background-color: ${LIGHT_GREY};
+  }
+`;
+
+const ConfirmDelete = styled(Button)`
+  margin: 10px;
+  padding-left: 10px;
+  &:hover {
+    background-color: ${LIGHT_GREY};
+  }
+`;
+
+interface CareEntryProps {
+  readonly activity: TreeCare;
+}
+
+function treeCareToMoment(activity: TreeCare) {
+  return moment(
+    `${activity.month} ${activity.day} ${activity.year}`,
+    'MMM Do YYYY',
+  );
+}
+
+const CareEntry: React.FC<CareEntryProps> = ({ activity }) => {
+  const [showEditForm, setShowEditForm] = useState<boolean>(false);
+  const [showDeleteForm, setShowDeleteForm] = useState<boolean>(false);
+  const [stewardshipFormInstance] = Form.useForm();
+
+  const id = Number(useParams<TreeParams>().id);
+  const dispatch = useDispatch();
+
+  function onFinishEditStewardship(values: RecordStewardshipRequest) {
+    const activities: ActivityRequest = {
+      date: values.activityDate.format('L'),
+      watered: values.stewardshipActivities.includes('Watered'),
+      mulched: values.stewardshipActivities.includes('Mulched'),
+      cleaned: values.stewardshipActivities.includes('Cleared Waste & Litter'),
+      weeded: values.stewardshipActivities.includes('Weeded'),
+    };
+    protectedApiClient
+      .editStewardship(activity.activityId, activities)
+      .then(() => {
+        message.success('Stewardship modified');
+        stewardshipFormInstance.resetFields();
+        dispatch(getSiteData(id));
+      })
+      .catch((err) =>
+        message.error(`Failed to record stewardship: ${err.response.data}`),
+      );
+  }
+
+  function onClickDeleteActivity() {
+    protectedApiClient
+      .deleteStewardship(activity.activityId)
+      .then(() => {
+        message.success('Stewardship Activity Deleted');
+        dispatch(getSiteData(id));
+      })
+      .catch((err) => {
+        message.error(
+          `Failed to delete stewardship activity: ${err.response.data}`,
+        );
+      });
+  }
+
+  const userIsAdmin: boolean = useSelector((state: C4CState) =>
+    isAdmin(state.authenticationState.tokens),
+  );
+
+  const userId: number = useSelector((state: C4CState) =>
+    getUserID(state.authenticationState.tokens),
+  );
+
+  const showButtons = userIsAdmin || userId === activity.userId;
+
+  return (
+    <>
+      <Entry>
+        <Row>
+          <Col span={5}>
+            <EntryDate>{activity.month + ' ' + activity.day}</EntryDate>
+          </Col>
+          <Col span={1} />
+          <Col span={13}>
+            <EntryMessage>{activity.message}</EntryMessage>
+          </Col>
+          <Col span={5}>
```

Comment:
🔧 For some reason, the delete button is in its own row when I visit the tree page:
![image](https://user-images.githubusercontent.com/33704204/214711475-079820f2-6d3a-4986-9988-3384dec0573d.png)

We can increase the width of this column but we should just increase the width of the entire stewardship activity container (which we can do by changing lines 277 and 299 in the `treePage` component).

Reply:
In addition to the above, we can also dynamically determine the width of the columns with the entries and buttons on line 154 in this component by doing something like `<Col span={showButtons ? 13 : 18}>` (similar thing for line 157) so that the entries can take up more space when we don't show the buttons. Let me know if I can clarify any of this!